share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md

```diff
@@ -1,52 +1,70 @@
 # How To Open a Homebrew Pull Request (and get it merged)
 
-The following commands are used by Homebrew's contributors to setup a fork of Homebrew's Git repository on GitHub, create a new branch and create a GitHub pull request of the changes in that branch.
+The following commands are used by Homebrew contributors to set up a fork of Homebrew's Git repository on GitHub, create a new branch, and create a GitHub pull request ("PR") of the changes in that branch.
 
-Depending on the change you want to make, you need to send the pull request to the corresponding repository. If you want to submit a change in Homebrew core code, you should open the pull request at [Homebrew/brew](https://github.com/Homebrew/brew). If you want to submit a change for a formula, you could open the pull request at [Homebrew core tap](https://github.com/Homebrew/homebrew-core) or any other [official taps](https://github.com/Homebrew) based the formula type.
+Depending on the change you want to make, you need to send the pull request to the appropriate one of Homebrew's main repositories. If you want to submit a change in Homebrew core code (the `brew` implementation), you should open the pull request on [Homebrew/brew](https://github.com/Homebrew/brew). If you want to submit a change for a formula, you should open the pull request on [the `homebrew/core` tap](https://github.com/Homebrew/homebrew-core) or another [official tap](https://github.com/Homebrew), based the formula type.
```

Comment:
> > “[…], you need to send the pull request to the appropriate **one of** Homebrew's main repositories.”
>
> Should be (?): “[…], you need to send the pull request to **one of** the appropriate Homebrew's main repositories.”

I disagree here. For any PR, there's exactly one repo which is appropriate for it, and part of the point of this paragraph is that you have to pick the right one.

The wording here *is* a little clunky, but I think the use of "one" is the problem, not the placement of "appropriate". But alternatives like "...to the appropriate repository amongst the several main Homebrew repositories" sound both stuffy and less clear to my ear.

> > “If you want to submit a change **in** Homebrew core code […]”
>
> Should be (?): “If you want to submit a change **to** Homebrew core code […]”

I think both work here, though "to" is more idiomatic. I'll change it.

> > “[…], based the formula type.”
>
> Should be (?): “[…], based **on** the formula type.”

Yep, thanks. Changing.